.github/workflows/process_test.yml

```diff
@@ -0,0 +1,45 @@
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, edited]
+  workflow_dispatch:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"        
```

Comment:
It is because each run produces the `level_3` datasets as artifacts, so doing this on three platforms (3.8, 3.9 and 3.10) produces three separate datasets... it might get a little bulky. 

So my thinking was the unit test workflow is a blanket test of compatibility, whereas the process test workflow is a "live" test that mimics our operational Python set-ups. 

An option could be that we don't produce the datasets as artifacts, and therfore don't accumulate bulky files. Runtime should be fine as each Python version run is executed simulataneously.